src/components/common/DiscountCodes.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div class="row">
+      <div class="col-sm-12">
+        <div id="appliedDiscounts"
+             class="text-right col-sm-12">
+          {{ $t('appliedDiscounts') }}:
+        </div>
+        <div v-for="discountInfo in cartLike.discountCodes"
+             :key='discountInfo.discountCode.id'
+             class="text-right col-sm-offset-6"
+             data-test="discount-code-list">
+          <div id="discountCodeName"
+               class="col-sm-8"
+               data-test="discount-code-name">
+            {{ discountInfo.discountCode.code }}
+          </div>
+          <div class="col-sm-3">{{ discountInfo.discountCode.name }}</div>
+          <RemoveDiscountCodeForm v-if="cartLike.__typename == 'Cart'"
```

Comment:
This should go to a computed property. It's okay to have small, simple and self-explained predicates in the template, but this one is not.